spec/lib/listen/directory_spec.rb

```diff
@@ -86,6 +86,20 @@
         end
       end
 
+      context 'when network share is disconnected' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.